splink/linker.py

```diff
@@ -2148,6 +2136,59 @@ def _compute_cluster_metrics(
 
         return df_cluster_metrics
 
+    def _compute_metrics_clusters(
+        self,
+        df_node_metrics: SplinkDataFrame,
+    ) -> SplinkDataFrame:
+
+        sqls = _size_density_centralisation_sql(
+            df_node_metrics,
+        )
+
+        for sql in sqls:
+            self._enqueue_sql(sql["sql"], sql["output_table_name"])
+
+        df_cluster_metrics = self._execute_sql_pipeline()
+        return df_cluster_metrics
+
+    # a user-facing function, which is currently 'private' (Beta functionality)
+    # while functionality is developed, as breaking changes may occur
+    def _compute_cluster_metrics(
+        self,
+        df_predict: SplinkDataFrame,
+        df_clustered: SplinkDataFrame,
+        threshold_match_probability: float,
+    ) -> Dict[str, SplinkDataFrame]:
+        """
+        Generates a table containing cluster metrics and returns a Splink dataframe
```

Comment:
Think this needs to be updated to match the `Returns` description

Reply:
Sorry, probably missing something obvious but not quite sure what you mean here?